src/timetrackings/timeTrackings.test.ts

```diff
@@ -0,0 +1,34 @@
+import TimeTrackings from './index';
+import { TimeTracking } from './types';
+import axios from 'axios';
+import AxiosMockAdapter from 'axios-mock-adapter';
+import RequestParams from '../utils/requestParams';
+
+describe('TimeTrackings', () => {
+  var timeTrackings: TimeTrackings = new TimeTrackings({});
+  var readPath: string = `${timeTrackings.getResourcePath()}/read`;
+  var mock = new AxiosMockAdapter(axios);
+  var result: Promise<TimeTracking[]>;
+
+  afterEach(() => {
+    mock.reset();
+  });
+
+  test('read', async () => {
+    mock.onGet(readPath).reply(200, { Success: true, NumResults: 1, Results: [{}] });
+    result = timeTrackings.read();
+    await result.then((result) => expect(result).toStrictEqual([{}]));
```

Comment:
Maybe better without the then
```suggestion
    result = await timeTrackings.read();
    expect(result).toStrictEqual([{}]);
```

Reply:
the "then" is needed, because we want to check on compile time that the result if of type Promise<Timetracking[]>